util/fetch_iso639-1_codelist.py

```diff
@@ -0,0 +1,30 @@
+"""
+Updates the ISO639-1 codelist from the Library of Congress TSV file.
+"""
+
+import csv
+import os
+import requests
+
+from helper import schema_dir
+
+
+if __name__ == '__main__':
+
+    codes = {}
+
+    response = requests.get('http://id.loc.gov/vocabulary/iso639-1.tsv')
+    response.raise_for_status()
+    content = response.content.decode('utf-8')
+    csvreader = csv.DictReader(content.splitlines(), delimiter='\t')
+    for row in csvreader:
+        code = row['code']
+        title = row['Label (English)']
```

Comment:
Some titles have repetitions, e.g. "Ndebele, North |  North Ndebele". Comma is also preferred for alternatives, instead of a vertical bar. I think the following will work. You'll need to `import re`

```suggestion
        titles = re.split(r' *\| *', row['Label (English)'])
        titles = ', '.join({' '.join(reversed(title.split(', '))): None for title in titles})
```

Reply:
Neat! Done in https://github.com/open-contracting/standard/pull/1316/commits/aca29923c8749cd573dc0433d44adfeec9e84564. I had to replace 'titles' with 'title' at the beginning of the second suggested row.